ethcore/res/ethereum/test-specs/berlin_test.json

```
@@ -110,13 +110,112 @@
 		"0000000000000000000000000000000000000009": {
 			"builtin": {
 				"name": "blake2_f",
-				"activate_at": "0x00",
+				"activate_at": "0x0",
 				"pricing": {
 					"blake2_f": {
 						"gas_per_round": 1
 					}
 				}
 			}
+		},
+		"000000000000000000000000000000000000000a": {
+			"builtin": {
+				"name": "bls12_381_g1_add",
+				"activate_at": "0x0",
+				"pricing": {
+					"0": {
+						"price": { "bls12_const_operations": { "price": 600 }}
+					}
```

```suggestion
					"bls12_const_operations": { "price": 600 }
```
also the same in this spec